docs/lab/results/result-formats.mdx

```diff
@@ -101,6 +103,120 @@ Interpretation is a string value that can be one of the following:
 3. `Interpretation.CRITICAL` - The result is outside of critical parameters.
     In this case, refer to the [critical values](/lab/workflow/lab-test-lifecycle#critical-results) section.
 
+#### LOINC
+
+As it is possible to test the same biomarkers across different labs, there must be a way to match results across labs. For this, we follow the LOINC standard.
```

Comment:
It's possible to test the same biomarkers across different labs. For these to match, we use the [LOINC](https://loinc.org) standard.